### PR TITLE
Update app-protection-policies.md

### PR DESCRIPTION
**Pull request by Leicham**
added a note to clarify the difference between public and custom apps

### CODE REVIEW

**Comment by PRMerger9** [comment]
@Leicham : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Erikre — APPROVED** [review]

**Comment by PRMerger12** [comment]
@Erikre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@Erikre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Erikre** [comment]
@Leicham - Thank you for your feedback and contribution. I'm adding this content now.

**Comment by Erikre** [comment]
#sign-off

File: memdocs/intune/apps/app-protection-policies.md [hunk 1]
```diff
@@ -84,7 +84,11 @@ When you create an app protection policy for iOS/iPadOS and Android apps, you fo
...
     | Custom apps | Click **Select custom apps** to select custom apps to target based on a Bundle ID. |
 
-    If you have selected individual apps, the apps will appear in the public and custom apps list.
+    The app(s) you have selected will appear in the public and custom apps list.
+    
+    > [!NOTE]
+    > **Public apps** are supported are apps from Microsoft and partners that are commonly used with Microsoft Intune. These Intune protected apps are enabled with a rich set of support for mobile application protection policies. For more information, see [Microsoft Intune protected apps](../apps/apps-supported-intune-apps.md). Custom apps are LOB apps that have been integrated with the Intune SDK or wrapped by the Intune App Wrapping Tool. For more information see [Microsoft Intune App SDK Overview](../developer/app-sdk.md) and [Prepare line-of-business apps for app protection policies](ne/developer/apps-prepare-mobile-application-management.md).
```
Comment (ShannonLeavitt) [inline memdocs/intune/apps/app-protection-policies.md:90]:
```suggestion
    > **Public apps** are supported are apps from Microsoft and partners that are commonly used with Microsoft Intune. These Intune protected apps are enabled with a rich set of support for mobile application protection policies. For more information, see [Microsoft Intune protected apps](../apps/apps-supported-intune-apps.md). Custom apps are LOB apps that have been integrated with the Intune SDK or wrapped by the Intune App Wrapping Tool. For more information see [Microsoft Intune App SDK Overview](../developer/app-sdk.md) and [Prepare line-of-business apps for app protection policies](../developer/apps-prepare-mobile-application-management.md).
```
Fixing link

**Comment by PRMerger6** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**